widget/playground/src/components/ExportConfigModal/ExportConfigModal.styles.tsx

```diff
@@ -79,10 +81,15 @@ export const Head = styled('div', {
   display: 'flex',
   padding: '$10 0',
   borderBottom: '1px solid $neutral300',
+  borderTop: '1px solid $neutral300',
+  margin: '$30 0',
```

Comment:
Can you use the divider component instead of it?

Reply:
Yes, I can. 
It's fixed